products/opensuse/main.pm

```diff
@@ -391,6 +391,7 @@ sub load_consoletests() {
             loadtest "console/http_srv.pm";
             loadtest "console/mysql_srv.pm";
             loadtest "console/dns_srv.pm";
+            loadtest "console/openvswitch.pm";
```

Comment:
Thanks for the test, but the place you put this is run in *every* scenario as acceptance test. What you wrote is a package test though, so please put this in the so called extra tests.